.github/MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,104 @@
+# DevDocs maintainer guidelines
+
+In general, the same [guidelines](https://devdocs.magento.com/guides/v2.3/contributor-guide/maintainers.html) for [`magento/magento2`](https://github.com/magento/magento2) maintainers apply to devdocs maintainers. However, there are some additional guidelines specific to devdocs that will help you as a maintainer.
+
+## General expectations
+
+- Self assign issues/pull requests (mostly pull requests)
+  - Review and approve or request changes
+  - Enforce the use of the issue/pull requests template
+  - Ask contributors to link to the code base to validate documentation updates when applicable
+  - Ask contributors not to contribute to v2.0 docs
+- If a maintainer creates a pull request, it should be reviewed by another maintainer or DevDocs staff member
+
+## Labels
+
+Labels are important because they help us identify pull requests and ensure that contributors receive points and recognition. See [devdocs awards and points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs-awards-and-points).
+
+Here is a brief summary of the most important labels:
+
+- `New Topic`: Entirely new documents
+- `Major Update`: Significant new info: new section in existing topic, etc.
+- `Technical`: Changes to technical content/code/processes/naming conventions (any change to technical content)
+- `Editorial`: Typos, grammatical inconsistencies, or minor rewrites
+- `small changes`: See [Small changes workflow](#small-changes-workflow)
+
+We also use version labels when appropriate.
+
+As a maintainer, we expect you to add or remove labels when appropriate.
+
+See https://github.com/magento/devdocs/labels for all labels and their descriptions.
+
+## Testing
+
+Currently, we only test internal links. In the future, we plan to expand tests to include markdown linting and spell checking.
+
+We use Jenkins to build, test, and deploy the devdocs site. Unfortunately, our Jenkins host is behind the corporate firewall (per Adobe security policy) and is not accessible to non-employees.
```

Comment:
Since there is no access to our CI/CD environment, I think there is no need to provide any information about it.
I suggest to change it to something like "We use private CI/CD stack and do not provide access to it."